src/General/Alert/AlertStyle.ts

```diff
@@ -31,16 +30,18 @@ export const AlertContainer = styled.div<AlertContainerProps>`
   transform: ${({ isVisible }) =>
     isVisible ? 'translate(-50%, 0%)' : 'translate(-50%, -100%)'};
   display: flex;
-  box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.12);
-  padding: 10px;
+  box-shadow: ${Shadow.down4};
+  padding: 12px;
   background-color: '#FFFAEA';
```

Comment:
Minor: looks like this background-color is unused, could you help to remove it? 🙏

Reply:
Done.